modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path

```diff
@@ -143,6 +143,16 @@
 	<td>//input[contains(@name,'neverExpire') and (@type='checkbox')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SELECT_CHECKBOX</td>
+	<td>//p[@title='${key_productName}']//../..//input[contains(@aria-label,'Compare')]</td>
+	<td></td> 
+</tr>
+<tr>
+	<td>ASSERT_MINI_COMPARE</td>
+	<td>//div[@class='mini-compare-item active']['${key_value}']//div[@class='mini-compare-thumbnail']</td>
+	<td></td> 
+</tr>
```

Comment:
Please rename this locator to `MINI_COMPARE_ITEM_ACTIVE_N` and move it to `CommerceFrontStore.path`.
```suggestion
<tr>
	<td>MINI_COMPARE_ITEM_ACTIVE_N</td>
	<td>//div[@class='mini-compare-item active'][${key_value}]//div[@class='mini-compare-thumbnail']</td>
	<td></td> 
</tr>
```